src/ui/controls/edit.h

```diff
@@ -132,8 +124,13 @@ class CEdit : public CControl
     bool        ReadText(std::string filename, int addSize=0);
     bool        WriteText(std::string filename);
 
-    void        SetMaxChar(int max);
-    int         GetMaxChar();
+    void        SetMaxChar(int max);
+    //! Get max characters that can be hold before truncation or if is Infinite edit actual allocated characters in m_text
+    int         GetMaxChar();
+    //! Has no character limit for edit
+    bool        IsInfinite();

```

Comment:
This should be removed along with SetInfinite(), external code doesn't really have any reason to care whether the the CEdit instance is infinite or not, if it does you can always use GetMaxChar() instead